### DISCUSSION TITLE
设置 sumatra pdf 不起作用

### DISCUSSION THREAD

**Discussion by weidongzhou1994** [James-Yu/LaTeX-Workshop#3154] [Q&A]
![image](https://user-images.githubusercontent.com/40510496/86099569-40c53100-baea-11ea-80dc-f8e9886e4f5b.png)

**Comment by tamuratak** [comment]
Fill out all the information required in the issue template. Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

If any progress, please notify.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)
